Yoast/yoast-components
# Implement SeoAnalysis and ReadabilityAnalysis components

## Summary

This PR can be summarized in the following changelog entry:

* Originally planned to create two components: SEO Analysis and Readability Analysis. However, this change has been reverted and now this PR creates support for using the smaller components which later be used to build the functionality of the original two planned components. 

## Relevant technical choices:

This PR became a little more involved than the issue would indicate.

* Added the `@wordpress/i18n` package to replace `react-intl`.
* Extracted a LanguageNotice component from ContentAnalysis.
* Removed the HelpText (and the LanguageNotice) from ContentAnalysis.
* Updated the HelpText & KeywordInput components.
* Replaced the `react-intl` with the `@wordpress/i18n` variant in the makeOutboundLink util.
* Updated KeywordExample to work correctly with new changes. 
* Added onChange to the KeywordInput props.
* Added 'Focus keyword:' label to KeywordInput.
* Tweaked design of components to better match proposed design: 
-- Changed the size of the collapsible header arrows.  
-- Made the header the "correct" blue: `blue_medium`, aka #1e8cbe.
-- The traffic light icons are resized to be slightly smaller
-- The blue headings should be slightly smaller: Same size as the text in the p element (13px) and the 'Focus keyword:'  heading.

## Related links

Sidebar design: https://github.com/Yoast/roadmap/issues/5#issuecomment-385635913
Keyword input for classic. See error message: https://github.com/Yoast/wordpress-seo/issues/9179#issuecomment-378182987

## Test instructions

This PR can be tested by following these steps:

* Normal pull: `git checkout stories/475-implement-seo-analysis-component`.
* Normal install: `yarn install`.
* Run the local dev server of yoast-components: `yarn start`.
* In the tab 'Content Analysis' the language notice should be at the top and the style changes should be present in the content analysis below. 
* In the tab 'Keyword' the HelpText should be visible above and the keyword input below. The keyword input should throw an error when a comma is entered. Also, there should be a 'Focus keyword' label.
* Checkout its sibling PR in WordPress-SEO: https://github.com/Yoast/wordpress-seo/pull/10438.
* Normal install: `yarn install`, `yarn link yoast-components`, `grunt build`
* Run the local dev server in WordPress-SEO: `yarn start`.
* Visit the edit-post page.
* Check the readability analysis in the metabox.
* Check if the language notice shows as it should.

Fixes #475
